Cargo.toml

```diff
@@ -38,6 +38,8 @@ netlink-packet-route = "0.13"
 netlink-packet-core = "0.4.2"
 fs2 = "0.4.3"
 netlink-sys = "0.8.3"
+netavark_proxy = { git = "https://github.com/containers/netavark-dhcp-proxy"} 
+tokio = { version = "1.0", features = ["rt"] }
```

Comment:
Any chance you can remove the tokio dep? You do not really use the async socket since you just block on it anyway. The amount of added dependencies here looks scary, building this PR shows me about a 4MB binary size increase.

Reply:
i think it will be required because the code is async